.config/nextest.toml

```diff
@@ -21,18 +21,26 @@ fail-fast = false
 # invocations of nextest happen.
 command = 'cargo run -p crdb-seed --profile test'
 
+[test-groups]
 # The ClickHouse cluster tests currently rely on a hard-coded set of ports for
 # the nodes in the cluster. We would like to relax this in the future, at which
 # point this test-group configuration can be removed or at least loosened to
 # support testing in parallel. For now, enforce strict serialization for all
 # tests with `replicated` in the name.
-[test-groups]
 clickhouse-cluster = { max-threads = 1 }
+# While most Omicron tests operate with their own simulated control plane, the
+# live-tests operate on a more realistic, shared control plane and test
+# behaviors that conflict with each other.  They need to be run serially.
+live-tests = { max-threads = 1 }
```

Comment:
1. Makes sense!
2. As mentioned in the DM, you can now filter out `omicron-live-tests` from [the default set](https://nexte.st/docs/running/#running-a-subset-of-tests-by-default). (You'll want to bump the required and recommended versions in this file, as well as the version used in CI, to 0.9.76)

Reply:
This is cool.  I'd prefer we do that in a separate PR.